eipw-lint/src/lints/markdown/relative_links.rs

```diff
@@ -38,13 +45,48 @@ impl<'e> Lint for RelativeLinks<'e> {
             .filter(|l| re.is_match(&l.address) && !exceptions.is_match(&l.address));
 
         for Link { line_start, .. } in links {
+            
+            let mut footer_label = String::new();
+            let mut footer = vec![];
+            let mut link_md = String::new();
+            
+            let line_with_address = ctx.line(line_start);
+            
+            match re.captures(line_with_address.as_bytes()) {
+                Some(caps) => {
+                    
+                    let line_link_address = str::from_utf8(&caps[0]).unwrap(); 
+                    write!(link_md, "`{}`",&line_link_address).unwrap(); 
```

Comment:
I don't see where you're looking for `eips.ethereum.org`. Issue #5 is specifically about suggesting EIP authors use `[EIP-1234](./eip-1234.md)` instead of something like `[EIP-1234](https://eips.ethereum.org/EIPS/eip-1234)`.

Reply:
Mine mistake, I was just solving the ` ...  `  suggestion, I will take care of this one.

As you can see below, finally passed the tests and the fmt and !  clippy ci.